src/shell.zig

```diff
@@ -522,9 +522,9 @@ pub fn exec_stdout_options(
     try child.collectOutput(&stdout, &stderr, options.output_bytes_max);
     const term = try child.wait();
     errdefer {
-        log.err("command failed", .{});
+        log.warn("command failed", .{});
         echo_command(argv.slice());
-        log.err("stdout:\n{s}\nstderr:\n{s}", .{ stdout.items, stderr.items });
+        log.warn("stdout:\n{s}\nstderr:\n{s}", .{ stdout.items, stderr.items });
     }
```

Comment:
These all are inside `errdefers`, so we are already propagating process-terminating error upwards here and, as such, `err` feels more appropriate.

This is in contrast to message bus where, although the source of errors is _also_ syscalls returning error code, there's a natural retry loop around the whole thing, which makes an individual error non-fatal.

No strong feelings here though!